doc/examples/qti_simulation_tutorial.py

```diff
@@ -0,0 +1,137 @@
+"""
+============================
+Q-SpaceTrajectory Simulation
+============================
+
+The most basic assumption of qti states that tissues can be denoted by multiple gaussian component.
+QTI can only handle a single gtab at a time, so in this tutorial we'll start by taking 2 gtabs then combine them into a single btensor. 
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+import math
+
+from dipy.reconst.tests.test_qti import  _anisotropic_DTD, _isotropic_DTD 
+from dipy.core.gradients import gradient_table 
+from dipy.core.sphere import disperse_charges, Sphere, HemiSphere
+from dipy.sims.voxel import multi_tensor
+
+"""
+We'll take the case of 2 gtabs and then combine them into a single btensor as for qti model, we need a single gtab.
+"""
+
+""" 
+Creating the first gtab 
+""" 
+
+n_pts = 20
```

Comment:
Let's make this tutorial more comparable to the QTI tutorial, so increase the number of directions here. Give a look to the bvecs, bvals and btens in the QTI tutorial (https://dipy.org/documentation/1.7.0/examples_built/07_reconstruction/reconst_qti/#sphx-glr-examples-built-07-reconstruction-reconst-qti-py). There you have 30 directions repeated for two b-values and repeated for two different encodings LTE and PTE.  So let's do that here.

Reply:
I.e. here change n_pts to 30